sparkling-water-mojo-scorer/example.py

```diff
@@ -0,0 +1,49 @@
+from pyspark.sql import SparkSession
+import argparse
+
+from pyspark.sql.types import StringType,FloatType
+from pyspark.sql.functions import monotonically_increasing_id
+
+def main():
```

Comment:
I might consider making this a class `SparklingWaterMojoScorer` or something. That way it is able to be passed around a little easier. Also, if its a bit more abstract users would be able to override certain functions for their own preferences?